File: algorithms/Codility-master/Codility-master/3_TimeComplexity/FrogJmp.js


// 100 %
function solution(X, Y, D) {
    // write your code in JavaScript (Node.js 8.9.4)
    if (Y <= X){
        return 0
    }
    let dis = Y - X
    if  (dis % D == 0){
        return Math.floor(dis / D)
    }
    else{
       return Math.floor(dis / D + 1)
    }
}

File: algorithms/Codility-master/Codility-master/3_TimeComplexity/FrogJmp.py

# 100% https://app.codility.com/demo/results/training34XJM2-PQU/
def solution(X, Y, D):
    # write your code in Python 3.6
    dis = Y - X
    if  dis % D == 0:
        return dis // D
    else:
       return dis // D + 1






       
